js/app.js

```diff
@@ -79,6 +81,20 @@ function headerRender() {
   trEl.appendChild(thEl);
   storeTable.appendChild(trEl);
 }
+function footerRender() {
+  var trEl = document.createElement('tr');
+  var thEl = document.createElement('th');
+  thEl.textContent = 'Column Totals';
+  trEl.appendChild(thEl);
+  for (var i = 0; i < hoursLong.length; i++) {
+    thEl = document.createElement('th');
+    thEl.textContent = columnTotals[i];
+    trEl.appendChild(thEl);
+  } thEl = document.createElement('th');
+  thEl.textContent = 'Total Daily Cookies';
```

Comment:
This last cell should have the total number of cookies sold in all stores for the entire day.